src/javascript/app/ng-wig/ng-wig.component.js

```diff
@@ -79,12 +79,8 @@ angular.module('ngWig')
         });
       });
 
-      $scope.$on('execCommand', (event, params) => {
+      this.execute = (command, options) => {
```

Comment:
do we actually need `execCommand ` and `execute `?
may be we can do 1 out of 2?